setup.py

```diff
@@ -37,6 +37,14 @@ def get_long_description():
     return readme_contents
 
 
+def walk_plugins_setup_py(action_name="UNNAMED", action="--help"):
+
+    for plugin in list(Path(os.getcwd()).rglob("./optional_plugins/*/setup.py")):
```

Comment:
Isn't it better to relate to the directory containing **this** file (__file__) instead of the current working directory?  IIUC, this won't operate correctly with a `cd .. && python3 avocado/setup.py develop`, but using `__file__` would.

Reply:
Good point @clebergnu !
I'm going to close this PR and made a new one. Once I have to work in that function I'd like as well to address the `make develop-external` target and that also requires not to hard code the path to the plugins.